image/Dockerfile

```diff
@@ -63,7 +65,7 @@ WORKDIR /build/hdf5-1.8.23/build-wasm
 RUN sed -i 's/js H5/js > H5/g' src/CMakeFiles/gen_hdf5-static.dir/build.make
 RUN emmake make -j`nproc` install
 
-FROM emscripten/emsdk:3.1.27
+FROM emscripten_base
```

Comment:
I like your goal here, but don't quite understand why you need to introduce an additional `emscripten_base` variable to achieve it.

Why not instead write `FROM qtbuilder` here?

Reply:
Because we want the `base` image, and add only the files we want using the minimal layer count.

If you use qtbuilder, you will also have all the temporary layers with build files and friends.